roles/servicetelemetry/defaults/main.yml

```diff
@@ -40,7 +40,7 @@ servicetelemetry_defaults:
       prometheus:
         enabled: true
         deployment_size: 1
-        scrape_interval: 10s
+        scrape_interval: 15s
```

Comment:
I wonder if this should really just be 30s.

I found a record of myself saying "I have no problem with those values" 9 days ago, and also the comment about "in _theory_ I'd rather sample twice as fast as a source, not half as fast", which is true, but was a bit of a joke meant to emphasize the very old SG behavior (and origin of the 5s/10s values in the code) where we were forced to sample half as fast because polling too fast would cause errors. We're not doing analog sampling, so Nyquist's Theorem doesn't apply here.  I think setting it to 30s to match everything else might be the simpler and better option.

Reply:
Done!